internal/webhook/module.go

```diff
@@ -175,6 +175,26 @@ func validateModuleLoaderContainerSpec(container kmmv1beta1.ModuleLoaderContaine
 		}
 	}
 
+	return validateModuleLoaderContainerBuildSpec(container.Build)
```

Comment:
We have Build definition both global (in ModuleLoaderContainerSpec) and in KernelMapping.  Currently you are verifying only the global one, and you are missing verification for Build definition in the KernelMappings